src/main/java/io/stargate/sgv2/jsonapi/api/exception/WebApplicationExceptionMapper.java

```diff
@@ -11,6 +13,13 @@ public class WebApplicationExceptionMapper {
 
   @ServerExceptionMapper
   public RestResponse<CommandResult> genericExceptionMapper(WebApplicationException e) {
+    // Return 405 for method not allowed and 404 for not found
+    if (e instanceof NotAllowedException) {
+      return RestResponse.status(RestResponse.Status.METHOD_NOT_ALLOWED);
+    }
+    if (e instanceof NotFoundException) {
+      return RestResponse.status(RestResponse.Status.NOT_FOUND);
+    }
```

Comment:
but here we are not adhering to the spec in case of the error? can we still create our body and report given statuses? seems strange now the mixup:

* some errors return `200` and a body from spec
* some errors return `4xx` and completely different body

Reply:
This is what is decided, refer #407

Reply:
OK, seems now there is no body at all.. Just status and empty response, I though this is using generic web application error body response..